### PR TITLE
feat: sorted rooms.json by name

### PR DESCRIPTION
**Pull request by lamontano**
For this MR I sorted rooms.json from app/config/temple/rooms.json. This were sorted by name using a linux command line tool called jq. Let me know if you have any questions or suggestions.

### CODE REVIEW

**Review by hndrewaall — CHANGES_REQUESTED** [review]
Thanks for helping out! 

You resolved the conflicts in the wrong direction (you effectively reverted the upstream changes, see the Files changed view). In this particular case, because we change these every week, this doesn't matter much. But it'd be good practice to try to get this so that it's just sorting the existing config in master.

File: authbot/authbot.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 SYNC_MAP = {
     "all-holidays-ticket-2022-pass": "AllHolidays2022",
     "spring-2022-benefactors-support-the-satanic-estate": "Benefactors",
-    "temple-7-05-22-the-devils-railroad-satanic-aid-in-crisis": "Temple5July22",
+    "temple-6-21-22-pride-vice-or-virtue": "Temple21June22",
```
Comment (hndrewaall) [inline authbot/authbot.py:21]: For example, you shouldn't be changing this file at all

**Comment by rene-grigory** [comment]
I reviewed this with Luis and he will be creating a new PR once he has his changes corrected. There was a little confusion with the branches and forks, but I think he's on the right track now. 

@hndrewaall : this is was an exercise to get Luis used to our process and Ada felt that having the temple rooms sorted would be nice for the caretakers since they don't typically mess with the room order in the temple.

**Comment by lamontano** [comment]
Closing this since this has been corrected in https://github.com/cabinweekend/jitsi-party/pull/12

**Pull request closed without merging**